packages/frontend/src/connectors/readonly/index.ts

```diff
@@ -2,14 +2,33 @@ import { JsonRpcProvider } from "@ethersproject/providers";
 import { Connector } from "wagmi";
 import { Address, Chain, ConnectorData, normalizeChainId } from "@wagmi/core";
 import { constants } from "ethers";
+import { ChainId } from "@carrot-kpi/sdk";
 
 export const READ_ONLY_CONNECTOR_ID = "readonly";
+const DEFAULT_CHAIN_ID = ChainId.GNOSIS; // 100
```

Comment:
We can remove the comment `// 100` here.

Reply:
Yes.

Reply:
fixed 43c7045c51411769a757cd05286ca9fc549dcc23